spec/lib/link_checker.rb

```diff
@@ -0,0 +1,133 @@
+require "./lib/link_checker"
+
+RSpec.describe LinkCheckerMethods do
+  describe "gather_urls method" do
+    it "should finds http urls in strings" do
+      result = LinkCheckerMethods.gather_urls "I've hidden an url http://www.example.com in this string"
+      expect(result).to eq(Set["http://www.example.com"])
+    end
+    it "should find https urls in strings" do
```

Comment:
I'm surprised the linter hasn't picked this up, but there should be an empty line between each test.